src/controllers/controllermanager.cpp

```diff
@@ -27,18 +27,18 @@
 #include "controllers/bulk/bulkenumerator.h"
 #endif
 
-namespace {
 // http://developer.qt.nokia.com/wiki/Threads_Events_QObjects
 
 // Poll every 1ms (where possible) for good controller response
 #ifdef __LINUX__
 // Many Linux distros ship with the system tick set to 250Hz so 1ms timer
 // reportedly causes CPU hosage. See Bug #990992 rryan 6/2012
-const int kPollIntervalMillis = 5;
+const mixxx::Duration ControllerManager::kPollInterval = mixxx::Duration::fromMillis(5);
 #else
-const int kPollIntervalMillis = 1;
+const mixxx::Duration ControllerManager::kPollInterval = mixxx::Duration::fromMillis(1);
```

Comment:
Why did you make this static instead of putting it into the anonymous namespace? is this needed elsewhere?

Reply:
I was using it before in HidController in the experimental approach that did not work with @codecat's controller. It should also be used in ControllerEngine/ControllerScriptInterfaceLegacy but that's beyond the scope of this PR, but I kept this small refactoring commit here.

Reply:
ok